src/components/shared/Common.js

```diff
@@ -29,15 +29,15 @@ export const H2 = styled.p({
   lineHeight: '11.52px',
 })
 
-export const DemoText = styled.p((props) => ({
-  fontFamily: 'Monaco',
-  fontSize: '18px',
-  color: '#FFFFFF',
-  fontWeight: '100',
-  margin: '0px',
-  padding: '10px',
-  ...props,
-}))
+export const DemoText = styled.p`
```

Comment:
another interesting thing I discovered while doing e2e is that if you use function e.g. `styled.p((props) => ...` you wont be able to add html attributes e.g. `data`. However, using the bellow or as per your pr allows to pass html attributes/props
```js
const a = styled.p`
  ...css
` 
```